src/Command/PluginLoadCommand.php

```diff
@@ -118,7 +118,7 @@ protected function modifyConfigFile(string $plugin, array $options): int
         } else {
             $array = var_export($config, true);
         }
-        $contents = '<?php' . "\n" . 'return ' . $array . ';';
+        $contents = '<?php' . PHP_EOL . 'return ' . $array . ';' . PHP_EOL;
```

Comment:
```suggestion
        $contents = '<?php' . "\n" . 'return ' . $array . ';' . "\n" ;
```
We use Unix's EOL `\n` for all our files, so should keep using the same here for consistency.

Reply:
So back to "\n" then? OK